titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 5 to 7_
```python
titanic = sns.load_dataset('titanic')

# Set the dark theme for the plot using matplotlib
```
test

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/25)._